endpoints/openrtb2/video_auction.go

```diff
@@ -198,6 +203,11 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 			labels.CookieFlag = pbsmetrics.CookieFlagYes
 		}
 	}
+	if labels.PubID == "UNKNOWN" && bidReq.Site != nil && bidReq.Site.Publisher != nil && bidReq.Site.Publisher.ID != "" {
+		labels.PubID = bidReq.Site.Publisher.ID
+	} else {
+		labels.PubID = "UNKNOWN"
```

Comment:
This is not right!  Line 209 can reset PubID to UNKNOWN when it already has a value.

Reply:
Copy/Paste error. Corrected